patches/tModLoader/Terraria/Wiring.cs.patch

```diff
@@ -116,6 +116,18 @@
  		switch (type) {
  			case 144:
  				HitSwitch(i, j);
+@@ -1157,8 +_,9 @@
+ 		}
+ 
+ 		if (type == 130) {
++			/* if (Main.tile[i, j - 1] == null || !Main.tile[i, j - 1].active() || (!TileID.Sets.BasicChest[Main.tile[i, j - 1].type] && !TileID.Sets.BasicChestFake[Main.tile[i, j - 1].type] && Main.tile[i, j - 1].type != 88)) { */
```

Comment:
Please place the `/*` and `*/` on separate lines so that the commented line shows up 'unchanged' in the patch